src/components/Tabs/Tabs.client.ts

```diff
@@ -262,22 +250,23 @@ class TabsComponent extends HTMLElement {
     // The switch-case will determine which tab should be marked as active
     // depending on the key that was pressed.
     let newTab;
-    switch (event.keyCode) {
-    case KEYCODE.LEFT:
-    case KEYCODE.UP:
+
+    switch (event.key) {
```

Comment:
Did `@astro/check` warn about this too? Perfectly happy with this. The tabs component was based on an implementation in a webdev article. So there's more improvements like this that can be made, but again 👍